connector/src/main/resources/salesforce/convertLead.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~  WSO2 Inc. licenses this file to you under the Apache License,
+ ~  Version 2.0 (the "License"); you may not use this file except
+ ~  in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing,
+ ~  software distributed under the License is distributed on an
+ ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+-->
+<template name="convertLead" xmlns="http://ws.apache.org/ns/synapse">
+    <parameter name="leadconvertrequests" description="The convert lead requests"/>
+    <sequence>
+        <property
+                expression="get-property('operation','salesforce.serviceUrl')"
+                name="uri.var.salesforce.url"/>
+        <payloadFactory>
+            <format>
+                <soapenv:Envelope
+                        xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
+                        xmlns:urn="urn:partner.soap.sforce.com">
+                    <soapenv:Header>
+                        <urn:SessionHeader>
+                            <urn:sessionId>$1</urn:sessionId>
+                        </urn:SessionHeader>
+                    </soapenv:Header>
+                    <soapenv:Body>
+                        <urn:convertLead/>
+                    </soapenv:Body>
+                </soapenv:Envelope>
+            </format>
+            <args>
+                <arg expression="get-property('operation','salesforce.sessionId')"/>
```

Comment:
What is the need to use `operation` context? Cannot we use `default` (synapse) context?

Reply:
Also instead of `get-property`, better  use `$ctx:` the new syntax if we can use synapse context.

Reply:
In the sf soap init they have used operation context

Reply:
We'll keep it for the moment. However, we'll evaluate